docs/guides/_include/define-env.rst

```diff
@@ -1,26 +1,45 @@
-Define the environment
-----------------------
+Define disk variables 
+---------------------
 
 For convenience and to reduce the likelihood of errors, set environment variables that refer to the devices that
 will be configured during the setup. Verify your target disk devices with ``lsblk``.
 
-First, define variables that refer to the disk and partition number that will hold boot files:
+.. tabs::
 
-.. parsed-literal::
+   .. group-tab:: Single Disk
 
-   export BOOT_DISK="\ |boot_disk|"
-   export BOOT_PART="\ |boot_part_no|"
-   export BOOT_DEVICE="${BOOT_DISK}${BOOT_PART}"
+      First, define variables that refer to the disk and partition number that will hold boot files:
 
-Next, define variables that refer to the disk and partition number that will hold the ZFS pool:
+      .. code-block::
 
-.. parsed-literal::
+        export BOOT_DISK="/dev/sda"
+        export BOOT_PART="1"
+        export BOOT_DEVICE="${BOOT_DISK}${BOOT_PART}"
 
-   export POOL_DISK="\ |pool_disk|"
-   export POOL_PART="\ |pool_part_no|"
-   export POOL_DEVICE="${POOL_DISK}${POOL_PART}"
+      Next, define variables that refer to the disk and partition number that will hold the ZFS pool:
 
-These definitions may be adjusted as needed to accommodate your system.
+      .. code-block::
 
+        export POOL_DISK="/dev/sda"
+        export POOL_PART="2"
+        export POOL_DEVICE="${POOL_DISK}${POOL_PART}"
+
+   .. group-tab:: Single Disk, external ESP
+
+      First, define variables that refer to the disk and partition number that will hold boot files:
+
+      .. code-block::
+
+        export BOOT_DISK="/dev/sdb"
+        export BOOT_PART="1"
+        export BOOT_DEVICE="${BOOT_DISK}${BOOT_PART}"
+
+      Next, define variables that refer to the disk and partition number that will hold the ZFS pool:
+
+      .. code-block::
+
+        export POOL_DISK="/dev/sda"
+        export POOL_PART="1"
+        export POOL_DEVICE="${POOL_DISK}${POOL_PART}"
```

Comment:
To further the DRY goal, you could move the text between the code blocks outside of the tabbed environment. This could be done with a general overview before a single code block that mashes together both var groups, or you could keep the current flow by using two separate groups. As long as changing tabs in one block syncs with with the other, this would provide a decent experience.